src/jobs/wallets/fetch-history-queue.ts

```diff
@@ -44,6 +44,17 @@ if (config.syncPacman) {
           QUEUE_NAME,
           `History Queue with transaction #${totalCount} of ${address} processing...`
         );
+        if (totalCount === 0) {
+          await walletHistoryQueue.addToQueue({
+            address,
+            batch: 0,
+            totalBatch: 0,
+            transactions: [],
+            workspaceId,
+            isWalletCached,
+          });
+          return true;
```

Comment:
I think we should log that we're done with processing before returning from this method.